bedrock/mozorg/views.py

```diff
@@ -154,12 +154,68 @@ def get_success_url(self):
         return reverse('mozorg.contribute.thankyou')
 
 
-def contribute_signup(request):
-    use_new_form = lang_file_has_tag('mozorg/contribute/index',
-                                     l10n_utils.get_locale(request),
-                                     '2015_signup_form')
-    view_class = ContributeSignup if use_new_form else ContributeSignupOldForm
-    return view_class.as_view()(request)
+class ContributeTasks(l10n_utils.LangFilesMixin, TemplateView):
+    template_name = 'mozorg/contribute/contribute-tasks.html'
+    variation_re = re.compile('^[1-4]$')
+
+    def get_context_data(self, **kwargs):
+        cxt = super(ContributeTasks, self).get_context_data(**kwargs)
+        variation = self.request.GET.get('variation', '4')
+        if self.variation_re.match(variation):
+            cxt['variation'] = variation
+        return cxt
+
+
+class ContributeTasksSurvey(l10n_utils.LangFilesMixin, FormView):
+    template_name = 'mozorg/contribute/contribute-tasks-survey.html'
+    task_re = re.compile('^[1-7]$')
+    form_class = ContributeTasksForm
+
+    def get_context_data(self, **kwargs):
+        cxt = super(ContributeTasksSurvey, self).get_context_data(**kwargs)
+        task = self.request.GET.get('task', '')
+        if self.task_re.match(task):
+            cxt['task'] = task
+        return cxt
+
+    def get_form_kwargs(self):
+        kwargs = super(ContributeTasksSurvey, self).get_form_kwargs()
+        kwargs['locale'] = l10n_utils.get_locale(self.request)
+        return kwargs
+
+    def get_success_url(self):
+        return reverse('mozorg.contribute.tasksthankyou')
+
+    def get_basket_data(self, form):
+        data = form.cleaned_data
+        return {
+            'email': data['email'],
+            'name': data['name'],
+            'country': data['country'],
+            'interest_id': 'dontknow',
+            'lang': form.locale,
+            'source_url': self.request.build_absolute_uri(),
+        }
+
+    def form_valid(self, form):
+        try:
+            basket.request('post', 'get-involved', self.get_basket_data(form))
+        except basket.BasketException as e:
+            if e.code == basket.errors.BASKET_INVALID_EMAIL:
```

Comment:
Yeah. Was late getting this in. Didn't find it a big deal. Can add one though if you want.